configs/piksiv3_defconfig

```diff
@@ -105,10 +105,14 @@ BR2_PACKAGE_HEALTH_DAEMON=y
 BR2_PACKAGE_RTCM3_PROTOCOL=y
 BR2_PACKAGE_CELL_MODEM_DAEMON=y
 BR2_PACKAGE_PIKSI_LEDS=y
-BR2_PACKAGE_LLVM_VANILLA=y
 BR2_PACKAGE_PIKSI_INS_REF=y
 BR2_PACKAGE_NETWORK_DAEMON=y
 BR2_PACKAGE_PORTS_DAEMON=y
 BR2_PACKAGE_PIKSI_INS=y
 BR2_PACKAGE_COMMON_INIT=y
 BR2_PACKAGE_QUERY_CONFIG=y
+BR2_PACKAGE_LLVM_VANILLA=y
+BR2_CCACHE=y
+BR2_CCACHE_DIR="$(BR2_EXTERNAL_piksi_buildroot_PATH)/buildroot/output/ccache"
+BR2_PACKAGE_METRICS_DAEMON=y
```

Comment:
`BR2_PACKAGE_METRICS_DAEMON=y` should be the only addition/change to this file for this PR